Installing-and-Configuring/OS/index.md

```diff
@@ -0,0 +1,133 @@
+
+  CONFIG_TPM_NO_LUKS_DISK_UNLOCK=y
+
+### Using an OS managed LUKS container (not heads)
+
+In this model heads is not involved in processes related to the decryption of data on disk. The OS manages keys and access to the LUKS containers.  
+
+This model is vulnerable to brute force attacks.
+
+### Security Dongle with LUKS root (not heads)
+
+In this model heads is not responsible for any of the decryption of data on disk.  You may use a security dongle (aka smartcard aka security token) to manage the passphrase for the root LUKS partition.  Like the TPM, the security dongle enforces limits which prevent brute force attacks.
+
+In contrast to [using TPM for decryption](#using-tpm-for-decryption-of-drives)--where a disk unlock passphrase is stored in the TPM--in this model the TPM is not used for LUKS decryption.  A LUKS disk unlock key/passphrase is encrypted and stored in a 'public' location such as /boot.  After heads boots the kernel a process which uses the security dongle decrypts the LUKS root partition.
+
+See also [TPM vs security dongle](#tpm-vs-security-dongle)
+
+TPM vs security dongle
+----
+
+Heads doesn't require a TPM but it is strongly suggested.  Heads does requires a USB Security dongle to enforce GPG operations such as /boot digest detach signing.  The Security dongle does not have to support HOTP but standard GPG is required.  see [/Prerequisites#usb-security-dongles-aka-security-token-aka-smartcard](/Prerequisites#usb-security-dongles-aka-security-token-aka-smartcard)
+
+How you configure the use of the security hardware depends on what you have and threat models you are working with.  Your system may not even have a TPM.  Both a TPM and a security dongle store private keys and allow for cryptographic operations.
+
+Heads will use the TPM for various operations and may be configured to use the TPM for disk unlock.  You have the option to use a security dongle for disk unlock instead of the TPM.  The significant differences between a TPM based Disk Unlock Key and USB Security dongle unlock are:
+
+* they are two different proprietary hardware chips
+* Security Dongles often have other features such as password management
+* the TPM is a generally soldered to the motherboard but the USB Security dongle is easily removable
+* since it is removable and may be used elsewhere, an eavesdropper sniffing the USB Security dongle's User PIN may gain access or be one step closer to accessing all systems protected by that hardware.  A TPM is embedded in a system so a compromise is limited to that system
+
+When using heads for a disk unlock key in the TPM, the Disk Unlock Key passphrase is valid until core components of OS are updated. This will force Heads to prompt the user to set a new boot default which gives the user the option of changing the Disk Unlock Key passphrase each time she signs files in /boot.
```

Comment:
When using heads for a disk unlock key in the TPM -> When using Heads for setting a Disk Unlock key in the TPM

It would be nice if all instances of the work heads was capitalized at Heads.
It would also be nice that all words that are Heads vocabulary were starting with uppercase letters so people understand that we are referring to the definitions. But that might be in another PR later on.

Reply:
capitalized Heads